### PR TITLE
correct the format of kafka sink

### PR DESCRIPTION
**Pull request by huangyuqi**
correct the format of kafka sink

### CODE REVIEW

File: docs/sink-configuration.md [hunk 1]
```diff
@@ -84,18 +84,19 @@ A combination of `insecure` / `caCert` / `auth` is not supported, only a single
...
 To use the kafka sink add the following flag:
 
-    --sink=kafka:<KAFKA_SERVER_URL>[?<OPTIONS>]
+    --sink="kafka:<?<OPTIONS>>"
 
 Normally, kafka server has multi brokers, so brokers' list need be configured for producer.
-So, we can set `KAFKA_SERVER_URL` to a dummy value, and provide kafka brokers' list in url's query string.
-Besides, the following options can be set in query string:
+So, we provide kafka brokers' list and topics about timeseries & topic in url's query string.
+Options should be set in query string, like this:
```
Comment (vishh) [inline docs/sink-configuration.md]: nit: Rephrase as `Options can be set in ...`

File: docs/sink-configuration.md [hunk 2]
```diff
@@ -84,18 +84,19 @@ A combination of `insecure` / `caCert` / `auth` is not supported, only a single
...
 
-* `timeseriestopic` - Kafka's topic for timeseries. Default: `heapster-metrics`
-* `eventstopic` - Kafka's topic for events. Default: `heapster-events`
+* `brokers` - Kafka's brokers' list
```
Comment (vishh) [inline docs/sink-configuration.md]: Why did you remove the defaults?

**Comment by vishh** [comment]
ok to test

File: docs/sink-configuration.md (hunk 2 above)
Comment (huangyuqi) [inline docs/sink-configuration.md]: sorry, i  have not noticed that you added the default topics. 
```golang
const (
	...
	timeSeriesTopic          = "heapster-metrics"
	eventsTopic              = "heapster-events"
)
```

**Comment by piosz** [comment]
LGTM. Thank you @huangyuqi for the contribution!

**Comment by huangyuqi** [comment]
thanks so much, @piosz   ^_^
